source/docs/articles/wordpress/wordpress-best-practices.md

```diff
@@ -0,0 +1,25 @@
+---
+title: WordPress Best Practices
+description: A list of suggestions for developing WordPress sites on Pantheon.
+keywords: wordpress, best practices, redis, wp-cfm, behat
+---
+
+This article provides suggestions, tips, and best practices for developing and managing WordPress sites on the Pantheon platform.
+
+* Use [Redis](/docs/articles/sites/installing-redis-on-wordpress/). Redis is an open-source, networked, in-memory, key-value data store that can be used as a drop-in caching backend for your WordPress site. Pantheon makes it super simple and you'll be able to catch a lot of database queries in WordPress.
+
+* Use [Grunt](http://gruntjs.com/) or [Gulp](http://gulpjs.com/) to aggregate JS/CSS on your side rather than relying on the server to do it for you. This helps speed up your workflow by minimizing redundant tasks.
+
+* Use [wp-cfm](https://pantheon.io/docs/articles/wordpress/wordpress-configuration-management-wp-cfm/). It lets you store settings from the `wp_options` table in Git and pull it into the database. A lot of WordPress stuff is option-heavy and you can spend a lot of time trying to figure out what you missed between environments. This is true for all WordPress sites, but especially helpful on Pantheon where you have at least three environments you will need to reconfigure every time.
+
+* Automate testing with [Behat](/docs/guides/automated-testing-wordpress-behat/). Adding automated testing into your development workflow will help you deliver higher quality WordPress sites.
+
+* Improve your site's security and SEO rankings by adding [CloudFlare Free SSL](/docs/articles/wordpress/add-cloudflare-free-ssl-to-wordpress-sites).
+
+* Use [JetBrains PhpStorm](docs/articles/wordpress/configuring-phpstorm-on-pantheon-for-wordpress), a commercial PHP IDE that can be configured to work with WordPress sites so you can easily build and maintain custom plugins.
```

Comment:
PhpStorm is great, but not sure about the wholesale endorsement (see https://en.wikipedia.org/wiki/Editor_war). Maybe remove this point or make it more general about choosing a good editor/IDE? Phpstorm is commercial, there are also many good free, alternatives like atom.io, netbeans, emacs, vi